### ISSUE TITLE
Subdivision questions

### ISSUE THREAD

**Issue by waizui** [trianglesplatting/triangle-splatting#38]
Thank you for your works!!
I noticed you use `sub2 = torch.stack([B, M_AB, M_BC], dim=1) ` in subdivsion of triangles, but this arrangment seems is clockwise, which flipped the normal. I wonder if it was done in purpose or just unintentionally?
``` 
def _update_params(self, selected_indices):
        selected_triangles_points = self._triangles_points[selected_indices]

        A = selected_triangles_points[:, 0, :]
        B = selected_triangles_points[:, 1, :]
        C = selected_triangles_points[:, 2, :]

        M_AB = (A + B) / 2
        M_AC = (A + C) / 2
        M_BC = (B + C) / 2

        sub1 = torch.stack([A, M_AB, M_AC], dim=1)
        sub2 = torch.stack([B, M_AB, M_BC], dim=1)
        sub3 = torch.stack([C, M_AC, M_BC], dim=1)
        sub4 = torch.stack([M_AB, M_AC, M_BC], dim=1)

        new_triangles_points = torch.cat([sub1, sub2, sub3, sub4], dim=0)

        new_features_dc = self._features_dc[selected_indices].repeat(4, 1, 1)
        new_features_rest = self._features_rest[selected_indices].repeat(4, 1, 1)
        new_opacities = self._opacity[selected_indices].repeat(4, 1)
        new_sigma = self._sigma[selected_indices].repeat(4, 1)
        new_mask = torch.ones_like(self._mask[selected_indices].repeat(4, 1))

        return new_triangles_points, new_features_dc, new_features_rest, new_opacities, new_sigma, new_mask
``` 
.

**Comment by heldJan** [comment]
Hi @waizui , thank you!

This was indeed unintentional, but it does not change anything. During rasterization, we always adjust the orientation of the normal so that it points toward the camera.
